app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java

```diff
@@ -815,6 +814,11 @@ public void onSnapshotReady(Bitmap snapshot) {
         LOG.trace("Exit");
     }
 
+    // Returns the absolute path of the parent folder for the IMAGE_FILE_FOLDER folder
+    private String getImageFolderParentFolder() {
+        return getApplicationContext().getExternalFilesDir(Environment.DIRECTORY_PICTURES).getPath() + File.separator;
```

Comment:
you can even make the `IMAGE_FILE_FOLDER` part of the file path returned here and call it `getImagePath()`